src/mjolnir/graphenhancer.cc

```diff
@@ -859,7 +859,7 @@ bool IsNextEdgeInternal(const DirectedEdge directededge,
                         GraphTileBuilder& tilebuilder,
                         GraphReader& reader,
                         std::mutex& lock,
-                        bool commercial_data) {
+                        bool infer_turn_channels) {
```

Comment:
should this be using `infer_internal_intersections` instead of turn channels?

Reply:
yeah i tested both at the same time.  let me fix